lit_nlp/components/metrics.py

```diff
@@ -226,6 +227,34 @@ def run_with_metadata(self,
     return self._metrics.run_with_metadata(indexed_inputs, model, dataset,
                                            model_outputs, margin_config)
 
+class ExactMatchMetrics(SimpleMetrics):
```

Comment:
@aryan1107 please sync this branch to the latest on `dev`. My [latest change](https://github.com/PAIR-code/lit/commit/0d8341d9f120359bec86c983c5618dd59bb6f591) updates Metrics considerably and I expect there will be merge conflicts here that we'l need to resolve. Happy to pair program to resolve those.